File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    L = 1e9
    U = 1e9
    R = 0
    D = 0
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if U>i and wallpaper[i][j] =='#':
                U = i
            if L > j and wallpaper[i][j] =='#':
                L=j    
            if R<j and wallpaper[i][j] =='#':
                R=j
            if D<i and wallpaper[i][j] =='#':
                D=i
                
    answer = [U,L,D+1,R+1]
    return answer

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    let minX = Infinity
    let minY = Infinity
    let maxX = 0
    let maxY = 0
    
    for(let i = 0; i < wallpaper.length; i++){
        const files = wallpaper[i].split("")
        for(let j = 0; j < files.length; j++){
            if(files[j]==="#"){
                minX = Math.min(minX, j)
                minY = Math.min(minY, i)
                maxX = Math.max(maxX, j+1)
                maxY = Math.max(maxY, i+1)
            }
        }
    }
    
    return [minY, minX, maxY, maxX]
}